daprdocs/content/en/reference/components-reference/supported-bindings/openai.md

```diff
@@ -228,6 +229,53 @@ The response body contains the following JSON:
 
 ```
 
+### Get Embedding API
+
+The `get-embedding` operations returns a vector representation of a given input that can be easily consumed by machine learning models and other algorithms.
+To perform a `get-embedding` operation, invoke the Azure OpenAI binding with a `POST` method and the following JSON body:
+
+```json
+{
+    "operation": "get-embedding",
+    "data": {
+        "deploymentId": "my-model",
+        "message": "The capital of France is Paris."
+    }
+}
+```
+
+The data parameters are:
+
+- `deploymentId` - string that specifies the model deployment ID to use.
+- `message` - string that specifies the text to embed.
+
+#### Example
+
+{{< tabs Linux >}}
+
+  {{% codetab %}}
```

Comment:
I don't think it makes a difference, but these don't need to be indented

Reply:
Removed indentation.